{"uuid":"fc57c95d-b48a-47a1-822b-a31d184d9a1f","name":"Scenario Outline: The user getting error message with invalid python code for Insertion page","children":["71064138-34ae-44c1-b666-c6d890e41266"],"befores":[{"name":"AppHooks.ApplicationHooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232815336,"stop":1706232815336}],"afters":[{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232815572,"stop":1706232815573},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232815750,"stop":1706232815750},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816237,"stop":1706232816237},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816237,"stop":1706232816237},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816550,"stop":1706232816550},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816764,"stop":1706232816764},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816795,"stop":1706232816795},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706232816800,"stop":1706232816800}],"start":1706232815336,"stop":1706232816800}